frontend/src/utils/privateRoute.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import { authService } from "../App";
+
+/**
+ * Ensures the user is logged-in before displaying them the route
+ */
+export const PrivateRoute = ({ component: Component, ...rest }) => {
```

Comment:
`rest` means the other props?

Reply:
Yes

Reply:
Is that a convention?
If not, would be better to call it `otherProps`
Yes, this is a detail

Reply:
Apparenlty yes